app/views/layouts/_proxy_notice.html.erb

```diff
@@ -0,0 +1,17 @@
+<div id="proxy-notice">
+  <div>
+    <% # TODO: When the interface is localized, this should be revised to only show the notice in the user's selected language. %>
+    <p><%= t(".text_html") %></p>
+    <p><%= t(".text_html", locale: :"zh-CN") %></p>
```

Comment:
This isn't currently working -- it just gives the English. I have tried the following:
```
    <%= t("layouts.proxy_notice.text_html", locale: :"zh-CN") %>
    <%= I18n.t("layouts.proxy_notice.text_html", locale: :"zh-CN") %>
    <%= I18n.t :"layouts.proxy_notice.text_html", locale: :"zh-CN" %>
    <%= I18n.t(".text_html", locale: :"zh-CN") %>
    <%= I18n.t ".text_html", locale: :"zh-CN" %>
    <%= I18n.t :text_html, locale: :"zh-CN" %>

    <%= t("layouts.proxy_notice.text_html", locale: "zh-CN") %>
    <%= I18n.t("layouts.proxy_notice.text_html", locale: "zh-CN") %>
    <%= I18n.t :"layouts.proxy_notice.text_html", locale: "zh-CN" %>
    <%= I18n.t(".text_html", locale: "zh-CN") %>
    <%= I18n.t ".text_html", locale: "zh-CN" %>
    <%= I18n.t :text_html, locale: "zh-CN" %>
```
I also tried putting it in `de.yml` just to see if the hyphenated name was the problem, but no luck.

Reply:
To clarify, some of those result in English, while others result in `translation missing: zh-CN.text_html translation`